### ISSUE TITLE
SQLite backed cache: Request for additional indexed fields #5

### ISSUE THREAD

**Issue by richard-cox** [rancher/rancher#48473]
As per https://github.com/rancher/rancher/issues/48256, this is the next batch.

### ADD
- networking.k8s.io.ingress
  - spec.rules.host
  - spec.ingressClassName
- service
  - spec.clusterIP

### REMOVE
- service
  - spec.targetPort - UI Bug this was never sortable
- networking.k8s.io.ingress
  - spec.rules - incorrect given vai does not support objects in all array entries (need specific path in array) 
  - spec.targetPort - incorrect c&p 

### CHANGE
- `"autoscaling", "v1", "HorizontalPodAutoscaler"`  --> `- "autoscaling", "v2", "HorizontalPodAutoscaler" `
  - see https://github.com/rancher/rancher/issues/48479#issuecomment-2555090389

**Comment by richard-cox** [comment]
This will be fixed by https://github.com/rancher/rancher/issues/48479

**Comment by moio** [comment]
I guess all is missing here is a steve bump to 0.5.1, right @tomleb ?

**Comment by tomleb** [comment]
Yes that's right, there's a PR opened here: https://github.com/rancher/rancher/pull/48638. I'll ping the reviewers to get it in.
